### PR TITLE
support lsq scheme reset

### PR DESCRIPTION
**Pull request by Jiang-Stan**
Fix bugs appear when feature is set to per_tensor_affine but its input data include nums less than 0.

### CODE REVIEW

**Comment by zhber** [comment]
LGTM

@PeiqinSun

File: sparsebit/quantization/quantizers/quant_descriptor.py [hunk 1]
```diff
@@ -48,6 +48,25 @@ def set_bit(self, bit):
         self._bit = bit
         self._qmin, self._qmax, self._type = self.calc_qmin_qmax(bit, self._scheme)
 
+    def reset_scheme(self, is_symmetric = None, is_perchannel = None):
```
Comment (PeiqinSun) [inline sparsebit/quantization/quantizers/quant_descriptor.py:51]: simplify code with "or"

File: sparsebit/quantization/quantizers/quant_descriptor.py [hunk 2]
```diff
@@ -48,6 +48,21 @@ def set_bit(self, bit):
...
+        self._scheme = {
+            torch.per_channel_affine: torch.per_channel_symmetric,
+            torch.per_channel_symmetric: torch.per_channel_affine,
+            torch.per_tensor_affine: torch.per_tensor_symmetric,
+            torch.per_tensor_symmetric: torch.per_tensor_affine,
+        }[self._scheme]
+        self._qmin, self._qmax, self._type = self.calc_qmin_qmax(
+            self._bit, self._scheme
+        )
+        self._ch_axis = self._set_channel_axis()
```
Comment (PeiqinSun) [inline sparsebit/quantization/quantizers/quant_descriptor.py:64]: why reset channel_axis?

File: sparsebit/quantization/quantizers/quant_descriptor.py [hunk 3]
```diff
@@ -48,6 +48,20 @@ def set_bit(self, bit):
         self._bit = bit
         self._qmin, self._qmax, self._type = self.calc_qmin_qmax(bit, self._scheme)
 
+    def set_symmetric(self, is_symmetric:bool):
```
Comment (PeiqinSun) [inline sparsebit/quantization/quantizers/quant_descriptor.py:51]:
```
self.is_symmetric = is_symmetric
self._scheme = {
      (True, True): torch.per_channel_symmetric,
      (True, False): torch.per_channel_affine,
      (False, True): torch.per_tensor_symmetric,
      (False, False): torch.per_tensor_affine,
}[(self.is_channelwise, self.is_symmetric)]
```

**Pull request merged**